serializer/custom_normalizer.rst

```diff
@@ -0,0 +1,106 @@
+.. index::
+   single: Serializer; Custom normalizers
+
+How to Create your Custom Normalizer
+====================================
+
+The :doc:`Serializer Component </components/serializer>` uses Normalizers
+to transform any data to an array.
+
+The Component provides several built-in normalizer that are described
+:doc:`in their own section </serializer/normalizer>` but you may want
+to use another structure that's not supported.
+
+Creating a new normalizer
+-------------------------
+
+Imagine you want add, modify, or remove some properties during the serialization
+process. For that you'll have to create your own normalizer. But it's usually
+preferable to let Symfony normalize the object, then hook into the normalization
+to customize the normalized data. To do that, we leverage the ObjectNormalizer::
+
+    namespace AppBundle\Serializer;
+
+    use App\Entity\Topic;
```

Comment:
you use `App` here, but 2 lines above you are in `AppBundle` namespace, maybe a mistake?

Reply:
yes, good catch :) thanks